src/mappings/mondo-sources-all-lexical-2.sssom.tsv

```diff
@@ -34806,8 +34806,8 @@ MONDO:0014949	developmental and epileptic encephalopathy, 47	skos:exactMatch	OMI
 MONDO:0014950	aortic aneurysm, familial thoracic 10	skos:closeMatch	Orphanet:91387	Familial thoracic aortic aneurysm and aortic dissection	semapv:LexicalMatching	oaklib	0.5	oio:hasDbXref	oio:hasDbXref	omim:617168
 MONDO:0014950	aortic aneurysm, familial thoracic 10	skos:closeMatch	Orphanet:91387	Familial thoracic aortic aneurysm and aortic dissection	semapv:LexicalMatching	oaklib	0.5	skos:exactMatch	oio:hasDbXref	omim:617168
 MONDO:0014950	aortic aneurysm, familial thoracic 10	skos:exactMatch	OMIM:153455	LOX	semapv:LexicalMatching	oaklib	0.9411764705882352	skos:exactMatch	skos:exactMatch	<http://linkedlifedata.com/resource/umls/id/c4284414>
-MONDO:0014951	Sotos syndrome 3	skos:closeMatch	Orphanet:821	Sotos syndrome	semapv:LexicalMatching	oaklib	0.5	oio:hasDbXref	oio:hasDbXref	omim:617169
-MONDO:0014951	Sotos syndrome 3	skos:closeMatch	Orphanet:821	Sotos syndrome	semapv:LexicalMatching	oaklib	0.5	skos:exactMatch	oio:hasDbXref	omim:617169
+MONDO:0014951	intellectual developmental disorder, autosomal recessive 74	skos:closeMatch	Orphanet:821	Sotos syndrome	semapv:LexicalMatching	oaklib	0.5	oio:hasDbXref	oio:hasDbXref	omim:617169
```

Comment:
Just trying to understand how this all works. Just a few questions.
1. It looks like `Sotos syndrome 3` and `Sotos syndrome` have a confidence match of `0.5?` But it seems like the strings are more similar than that. Why 0.5?
2. It looks like the label on `MONDO:0014951` changed from `Sotos syndrome 3` to `intellectual developmental disorder, autosomal recessive 74`, but the confidence didn't change. Why?
3. It looks like the final column is called `match_string`, but the values in these columns are sometimes labels, and sometimes they are CURIEs. In this case, it's `omim:617169`. Just wondering how that comes into play as a match string?

Reply:
1. 0.5 is a placeholder (I think) for all matches that are not high confidence.
2. Just a label update probably came from the input to `lexmatch`
3. `oio:hasDbXref` So OMIM <=> Orphanet relation exists and Mondo <=> OMIM relation exists. Hence match string = `omim:xxxx`